lib/ddr/managers/derivatives_manager.rb

```diff
@@ -0,0 +1,119 @@
+require "resque"
+
+module Ddr
+  module Managers
+    class DerivativesManager < Manager
+
+      SCHEDULE_LATER = :later
+      SCHEDULE_NOW = :now
+      SCHEDULES = [ SCHEDULE_LATER, SCHEDULE_NOW ]
+
+      ACTION_DELETE = "delete"
+      ACTION_GENERATE = "generate"
+
+      def update_derivatives schedule=SCHEDULE_LATER
```

Comment:
Although I haven't been consistent about this ... I think now that it's better to use parentheses in method signatures.  At least I find them easier to read at a glance.